src/mozilla-traffic-cop.js

```diff
@@ -171,12 +215,12 @@ Mozilla.TrafficCop.prototype.cookieExpiresDate = function(date) {
 /*
  * Checks to see if user is currently viewing a variation.
  */
-Mozilla.TrafficCop.prototype.isVariation = function(queryString) {
+Mozilla.TrafficCop.isVariation = function(variations, queryString) {
```

Comment:
Is it worth calling out that this only works for variations that are redirects? Maybe we could make that more explicit in the name?